packages/catalog-search/src/SearchBox.jsx

```diff
@@ -31,6 +41,8 @@ export const SearchBoxBase = ({
   index,
   filters,
   enterpriseSlug,
+  suggestionSubmitOverride,
+  disableSuggestionRedirect,
```

Comment:
hmm looks like disableSuggestionRedirect is a pass through prop that is really only used by other components. In that case do you thin it makes sense to set it as 

```
disableSuggestionRedirect = typeof suggestionSubmitOverride === 'function'
```

then pass it along to the other components? This way you will end up with only one new prop here instead of two

Just a thought to see if we can further simplify things.

Reply:
also may be easier for the authors using this component to think of one concept as one prop, rather than two...